Source/WebKit/UIProcess/SubframePageProxy.cpp

```diff
@@ -104,6 +104,7 @@ void SubframePageProxy::decidePolicyForResponse(WebCore::FrameIdentifier frameID
 
 void SubframePageProxy::didCommitLoadForFrame(WebCore::FrameIdentifier frameID, FrameInfoData&& frameInfo, WebCore::ResourceRequest&& request, uint64_t navigationID, const String& mimeType, bool frameHasCustomContentProvider, WebCore::FrameLoadType frameLoadType, const WebCore::CertificateInfo& certificateInfo, bool usedLegacyTLS, bool privateRelayed, bool containsPluginDocument, WebCore::HasInsecureContent hasInsecureContent, WebCore::MouseEventPolicy mouseEventPolicy, const UserData& userData)
 {
+    m_process->didCommitProvisionalLoad();
```

Comment:
This is needed to that we don't end up using the same process because of:

```
    if (!sourceProcess->hasCommittedAnyProvisionalLoads()) {
        tryPrewarmWithDomainInformation(sourceProcess, targetRegistrableDomain);
        return { WTFMove(sourceProcess), nullptr, "Process has not yet committed any provisional loads"_s };
    }
```

now that we pass the right source process to `WebProcessPool::processForNavigation`